cdk/src/open-data-platform/data-plane/data-import/write-demographic-data.handler.ts

```diff
@@ -219,7 +293,43 @@ class DemographicsTableRowBuilder {
     return this;
   }
 
-  build(): DemographicsTableRow {
-    return this._row;
+  geom(geom: string): DemographicsTableRowBuilder {
+    this._row.geom = geom;
+    return this;
+  }
+
+  // TODO clarify these are the right types (double?)
+  build(): SqlParametersList {
+    return [
+      {
+        name: 'census_geo_id',
+        value: { stringValue: this._row.census_geo_id.toString() },
+      },
+      {
+        name: 'total_population',
+        value: { doubleValue: this._row.total_population },
+      },
+      {
+        name: 'under_five_population',
+        value: { doubleValue: this._row.under_five_population },
+      },
+      {
+        // TODO(kailamjeter) make this a percentage
```

Comment:
Do we want this as a percentage? I thought we were doing raw data here and aggregating in views?

Reply:
Same here - old commit.